upup/pkg/fi/cloudup/new_cluster.go

```diff
@@ -304,6 +309,20 @@ func NewCluster(opt *NewClusterOptions, clientset simple.Clientset) (*NewCluster
 	return &result, nil
 }
 
+func setupRuntimes(opt *NewClusterOptions, cluster *api.Cluster) {
```

Comment:
I wonder if not we should just set containerd here. It means we cannot change the implicit default later on, but I think changing the default would be very far into the future, and likely will involve changing default for new clusters first anyway.

Reply:
We can't. We just remove something similar from here.
It is not a good idea to do this, and materialize it for all new clusters. If you really want it enabled by default maybe consider enabled when not set explicitly.

Reply:
Well we can. It is just a matter of setting it here.

We cannot enable  nvidia on existing clusters though, so this would be the most helpful. Or decide it must always be enabled explicitly.

Reply:
I don't mind enabling it explicitly. I don't mind enabling it on existing clusters either.
I think it is a really low risk move, considering you actually have to use GPU instances for this.

Reply:
If a cluster is using GPU operator, it may break things. It could break things on prebaked images if they expect other drivers or other paths as well. I am not sure it is that safe to just enable it on existing clusters and I fear those who already install things their own way will not pick up this change from the release notes.

Reply:
My idea with adding the nvidia conf here was to make it "just work" for those using `kops create cluster`. Those using specs would have to enable this explicitly in all cases.

It is not a must to keep this particular commit.

It could also be an option to add a flag, but ... seems a bit much.

Reply:
OK, then enabling explicitly is the way, from my point of view.
If we get feedback that "it's hard to use/enable" we can discuss more.